__tests__/app.tests.js

```diff
@@ -36,3 +36,15 @@ describe("/api/topics", () => {
       });
   });
 });
+describe("GET api ", () => {
+  test("GET - API should return a json with all the available end points ", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        console.log(res.body);
```

Comment:
In future don't leave console logs left in commited code

Reply:
Ah maybe you're using this to prove that the correct JSON is coming back. In which case see my second point, you can make a specific test for it 👍